lib/ESP8266PWM/src/core_esp8266_waveform_pwm.cpp

```diff
@@ -0,0 +1,711 @@
+/* esp8266_waveform imported from platform source code
+   Modified for WLED to work around a fault in the NMI handling,
+   which can result in the system locking up and hard WDT crashes.
+*/
```

Comment:
Please add a URL or exact release name for the original file. This might save a few hair later, in case we want to follow an updated arduino-8266 release...

It might also help if you mark your changes with a comment like ` //  willmiles restore lost processor context`, or put your changes between `// begin patch` ..... `// end patch`

Reply:
Done.  Thanks for the review!